server/plugin/api_test.go

```diff
@@ -632,7 +680,8 @@ func TestHandleEndPoll(t *testing.T) {
 			p := setupTestPlugin(t, api, store)
 
 			w := httptest.NewRecorder()
-			r := httptest.NewRequest("POST", fmt.Sprintf("/api/v1/polls/%s/end", testutils.GetPollID()), bytes.NewReader(test.Request.ToJson()))
+			r := httptest.NewRequest(http.MethodPost, fmt.Sprintf("/api/v1/polls/%s/end", testutils.GetPollID()), bytes.NewReader(test.Request.ToJson()))
```

Comment:
`TestHandleAddOptionDialogRequest` doesn't have a test for `not-authorized`.   Are there any reasons?

Reply:
Nope, I just forgot it. Added a test for this.